### ISSUE TITLE
Illegal instruction

### ISSUE THREAD

**Issue by dabasmoti** [UKPLab/sentence-transformers#485]
Hi,
I have Illegal instruction error 
I am running on :
Centos 7
pytorch - '1.6.0+cpu'
transformers - 3.3.1
sentence_transformers - 0.3.7
gcc (GCC) 4.8.5 20150623 (Red Hat 4.8.5-39)
Any suggestions?

**Comment by nreimers** [comment]
Could you post the complete exception message. And also the code you try to run.

**Comment by dabasmoti** [comment]
@nreimers 
Hi,
The exception is only 
"Illegal instruction "
and then the kernel is died?
```
from sentence_transformers import SentenceTransformer

model = SentenceTransformer('xlm-r-100langs-bert-base-nli-stsb-mean-tokens')

lines = ['A man is eating food.',
          'A man is eating a piece of bread.',
          'A man is eating pasta.',
          'The girl is carrying a baby.',
          'The baby is carried by the woman',
          'A man is riding a horse.',
          'A man is riding a white horse on an enclosed ground.',
          'A monkey is playing drums.',
          'Someone in a gorilla costume is playing a set of drums.',
          'A cheetah is running behind its prey.',
          'A cheetah chases prey on across a field.']

sentence_embeddings = model.encode(lines)
```

I am using old machine, maybe that the reason?

**Comment by nreimers** [comment]
Transformer require quite a lot of memory. You could try the distilbert models and also set the batch size to only a single sentence and see if it works then.

**Comment by dabasmoti** [comment]
@nreimers 
I have 32Gb and 16 cpu's
I will try to run on single sentence and see the results

**Comment by dabasmoti** [comment]
@nreimers 
I tried on single sentence , the error stand the same.

**Comment by dabasmoti** [comment]
@nreimers 
I reinstalled the env using pytorch 1.4(install using conda)
sentence-transformer(pip)
and i have different error:
```
from sentence_transformers import SentenceTransformer

model = SentenceTransformer('xlm-r-100langs-bert-base-nli-stsb-mean-tokens')
#model = SentenceTransformer('distilbert-base-nli-mean-tokens')

lines = ['A man is eating food.']

sentence_embeddings = model.encode(lines)
print(sentence_embeddings)

Traceback (most recent call last):
  File "/root/conda/envs/trans/lib/python3.7/site-packages/transformers/modeling_utils.py", line 927, in from_pretrained
    state_dict = torch.load(resolved_archive_file, map_location="cpu")
  File "/root/conda/envs/trans/lib/python3.7/site-packages/torch/serialization.py", line 527, in load
    with _open_zipfile_reader(f) as opened_zipfile:
  File "/root/conda/envs/trans/lib/python3.7/site-packages/torch/serialization.py", line 224, in __init__
    super(_open_zipfile_reader, self).__init__(torch._C.PyTorchFileReader(name_or_buffer))
RuntimeError: version_ <= kMaxSupportedFileFormatVersion INTERNAL ASSERT FAILED at /opt/conda/conda-bld/pytorch_1579022030672/work/caffe2/serialize/inline_container.cc:132, please report a bug to PyTorch. Attempted to read a PyTorch file with version 3, but the maximum supported version for reading is 2. Your PyTorch installation may be too old. (init at /opt/conda/conda-bld/pytorch_1579022030672/work/caffe2/serialize/inline_container.cc:132)
frame #0: c10::Error::Error(c10::SourceLocation, std::string const&) + 0x47 (0x7fe9f127a627 in /root/conda/envs/trans/lib/python3.7/site-packages/torch/lib/libc10.so)
frame #1: caffe2::serialize::PyTorchStreamReader::init() + 0x1f5b (0x7fe96e604cbb in /root/conda/envs/trans/lib/python3.7/site-packages/torch/lib/libtorch.so)
frame #2: caffe2::serialize::PyTorchStreamReader::PyTorchStreamReader(std::string const&) + 0x64 (0x7fe96e605ed4 in /root/conda/envs/trans/lib/python3.7/site-packages/torch/lib/libtorch.so)
frame #3: <unknown function> + 0x69e366 (0x7fe9f1cb3366 in /root/conda/envs/trans/lib/python3.7/site-packages/torch/lib/libtorch_python.so)
frame #4: <unknown function> + 0x26b876 (0x7fe9f1880876 in /root/conda/envs/trans/lib/python3.7/site-packages/torch/lib/libtorch_python.so)
<omitting python frames>
frame #52: __libc_start_main + 0xf5 (0x7fe9fb9ee555 in /lib64/libc.so.6)


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "test.py", line 4, in <module>
    model = SentenceTransformer('xlm-r-100langs-bert-base-nli-stsb-mean-tokens')
  File "/root/conda/envs/trans/lib/python3.7/site-packages/sentence_transformers/SentenceTransformer.py", line 106, in __init__
    module = module_class.load(os.path.join(model_path, module_config['path']))
  File "/root/conda/envs/trans/lib/python3.7/site-packages/sentence_transformers/models/Transformer.py", line 101, in load
    return Transformer(model_name_or_path=input_path, **config)
  File "/root/conda/envs/trans/lib/python3.7/site-packages/sentence_transformers/models/Transformer.py", line 30, in __init__
    self.auto_model = AutoModel.from_pretrained(model_name_or_path, config=config, cache_dir=cache_dir)
  File "/root/conda/envs/trans/lib/python3.7/site-packages/transformers/modeling_auto.py", line 589, in from_pretrained
    pretrained_model_name_or_path, *model_args, config=config, **kwargs
  File "/root/conda/envs/trans/lib/python3.7/site-packages/transformers/modeling_utils.py", line 930, in from_pretrained
    "Unable to load weights from pytorch checkpoint file. "
OSError: Unable to load weights from pytorch checkpoint file. If you tried to load a PyTorch model from a TF 2.0 checkpoint, please set from_tf=True.  
```

**Comment by nreimers** [comment]
Hi @dabasmoti 
Pytorch introduced a new file format for storing models in version 1.6.

The newer models (like the multilingual models) are in this file format. Hence, I can recommend to update to pytorch 1.6. Then it should work.

**Comment by dabasmoti** [comment]
@nreimers 
The error in the first place was with pytorch version 1.6,
look at [this](https://github.com/pytorch/pytorch/issues/43300)
so i decided to downgrade

**Comment by nreimers** [comment]
Do you need the multilingual models? The most other English models are still stored in the old pytorch file format.

**Comment by dabasmoti** [comment]
yep,
I am working with Hebrew :)

**Comment by malfet** [comment]
@dabasmoti can you try if it was fixed in PyTorch-1.7 release candidate, see https://download.pytorch.org/whl/test/cpu/torch_test.html
